app/models/user.rb

```diff
@@ -1,2 +1,9 @@
 class User < ApplicationRecord
+  has_many :posts
+  has_many :comments
+  has_many :likes
+
+  def rent_posts(limit = 3)
+    posts.order(created_at: :desc).limit(limit)
+  end
```

Comment:
- [ ] To follow a naming convention, it would be great to call your method `recent_post`
- [ ] Remember that to use the limit method, it's not necessary to receive a parameter in the method. You should add the number of posts that you want to limit in the last sentence of your code. Like `post.order('created_at Desc').limit(3)`